openapi.yml

```diff
@@ -1,8 +1,31 @@
 openapi: "3.0.0"
 info:
   title: "IO onboarding PA API"
-  description: "The backend used by the onboarding portal for public administrations of the IO project."
+  x-summary: >-
+    Public Administriations onboarding APIs for the IO project.
+  description: |-
+      The backend used by the onboarding portal for public administrations of the IO project.
+
+      The website for the project is [io-website].
+      Privacy Policy is [privacy].
+
+      Development happens on github.
+
```

Comment:
I'd remove a blank line here

Reply:
I just made it for you to complete with all the required informations. As I don't know them
I put there some example informations. Edit this PR and I'll provide you feedback.